test/images/scripts/fedora-25.setup

```diff
@@ -76,6 +76,12 @@ maybe() { if type "$1" >/dev/null 2>&1; then "$@"; fi; }
 # For the D-Bus test server
 maybe firewall-cmd --permanent --add-port 8765/tcp
 
+# If the main VM run a dnsmasq instance
+for x in $(seq 1 9); do
+    nmcli connection add type ethernet con-name machine1 ifname "*" mac ee:00:0a:00:00:$x ip4 10.0.0.$x/24
+done
```

Comment:
This just adds client-side interfaces in NM, not a DNS server. Fedora doesn't default to using a client-side dnsmasq local DNS resolver. The comment could be clarified, also why it needs a dnsmasq?

Depending on what the purpose is, I figure this needs to be replicated to fedora-{26,testing}.setup? What about debians, Atomics, etc?